### DISCUSSION TITLE
Got my server working after 0.211.7 patch

### DISCUSSION THREAD

**Discussion by proofquantu** [lloesche/valheim-server-docker#505] [Show and tell]
My server runs with BepInEx and after the 0.211.7 update the server won't start.
Thank to the forward thinking of the dev team I was able to adjust the steamCMD command in the env file to get it working.

STEAMCMD_ARGS='-beta default_old'

Thanks for such a well thought out container implementation.

**Comment by tomekduda** [comment]
This should enable us to participate in [Mistlands beta](https://valheim.com/news/mistlands-public-test/) too, right?

`-beta public-test`

**Comment by Ozwald1** [comment]
Cat't get working, not even if add -betapassword "yesimadebackups".. :slightly_frowning_face:

**Reply by Concretv** [reply]
> Cat't get working, not even if add -betapassword "yesimadebackups".. 🙁

The correct arguments to run the mistlands beta are : STEAMCMD_ARGS='app_update 896660 -beta public-test -betapassword yesimadebackups'

**Reply by syntaxTerr0r** [reply]
Thank you for this!

**Reply by Ozwald1** [reply]
Ok, thanks finally managed. For some reason I had to add validate.. 

STEAMCMD_ARGS='app_update 896660 -beta public-test -betapassword yesimadebackups validate'
